$OpenBSD: patch-src_client_util_util-date_vala,v 1.1 2017/04/13 07:10:21 jasper Exp $

Fix build with vala 0.36:
error: Argument 1: Cannot convert from `GLib.TimeSpan' to `int'

--- src/client/util/util-date.vala.orig	Thu Apr 13 08:55:09 2017
+++ src/client/util/util-date.vala	Thu Apr 13 08:56:20 2017
@@ -164,7 +164,7 @@ private string pretty_print_coarse(CoarseDate coarse_d
             return _("Now");
         
         case CoarseDate.MINUTES:
-            return ngettext("%dm ago", "%dm ago", (ulong) (diff / TimeSpan.MINUTE)).printf(diff / TimeSpan.MINUTE);
+            return ngettext("%dm ago", "%dm ago", (ulong) (diff / TimeSpan.MINUTE)).printf((int) (diff / TimeSpan.MINUTE));
         
         case CoarseDate.HOURS:
             int rounded = (int) Math.round((double) diff / TimeSpan.HOUR);
